kernel/internal/process.go

```diff
@@ -27,7 +27,7 @@ import (
 
 func NewProcess(name string, cmd ...string) *Process {
 	return &Process{
-		Cmd:       exec.Command(name, cmd...),
+		Cmd: exec.Command(name, cmd...),
 		outStream: make(chan []byte),
 		errStream: make(chan []byte),
```

Comment:
Something going on with your `go fmt`? Why did we unformat this?

Reply:
Not sure. Didn't do this on purpose. I also went back and formatted the file and it reverted this change.

I'll keep a lookout just in case I have a setting wrong.